recipe/meta.yaml

```diff
@@ -133,6 +133,9 @@ outputs:
         - pip check
 
   - name: pytorch-cpu   # [cuda_compiler_version == "None"]
+    test:
+      imports:
+        - torch
```

Comment:
Please remove these 3 lines.

Reply:
> For recipe:
 It looks like the 'pytorch-cpu' output doesn't have any tests.

Just curious, why does the linter flag this?

Reply:
Linter doesn't understand selectors. It's wrong to have this here. Please remove them.

Reply:
Done

Reply:
Thanks. FYI, the two lines `- name: pytorch-cpu` and `- name: pytorch-gpu` are mutually exclusive and the lines below them should apply to both.